internal/backend/remote-state/oci/backend_test.go

```diff
@@ -0,0 +1,472 @@
+package oci
```

Comment:
```suggestion
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: BUSL-1.1

package oci
```

This should address the remaining CI failure